file-formats/dsfi/type/zif_aff_dsfi_v1.intf.abap

```diff
@@ -0,0 +1,43 @@
+INTERFACE zif_aff_dsfi_v1
+  PUBLIC.
+
+
+  "! <p class="shorttext">Engine</p>
+  "! Engine
+  "! $values {@link if_aff_dsfi_v1.data:co_engine}
+  TYPES: ty_engine TYPE c LENGTH 3.
+
+  CONSTANTS:
+    BEGIN OF co_engine,
+      "! <p class="shorttext">Analytical Engine</p>
+      "! Analytical engine
+      analytical_engine TYPE ty_engine VALUE 'ANA',
```

Comment:
Do you expect to have more possible values for "engine" in the future?

Reply:
yes, if remember correctly there will be two more engines.